lib/src/publication/remote.dart

```diff
@@ -227,6 +231,12 @@ class RemoteTrackPublication<T extends RemoteTrack>
     sendUpdateTrackSettings();
   }
 
+  Future<void> setVideoFPS(int newValue) async {
```

Comment:
a bit of comment here would be helpful. Stating that this is setting the desired FPS, server will do its best to return FPS close to this. It's only supported for video codecs that support SVC currently.